ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
+    > random(1000)
+    < 882.4784
+    ~ \d+\.\d+
     > sum(1..1000, random(5))
-    < 2464.8003929607607
+    < 2464.8004
     ~ 2\d\d\d\.\d+
```

Comment:
I do not see why the result should start with digit 2. I believe `sum(1..1000, random(5))` should be equivalent to `random (5000)`.

In this case, the regexp should be `\d{1,4}(\.\d+)?`. Otherwise, please explain.